core/sdk-core/src/main/java/software/amazon/awssdk/core/exception/SdkServiceException.java

```diff
@@ -225,6 +231,17 @@ public Builder writableStackTrace(Boolean writableStackTrace) {
             return this;
         }
 
+        @Override
+        public Builder numAttempts(Integer numAttempts) {
```

Comment:
I don't remember why, but it seems we may need to add this new method to every generated exception class. 

https://github.com/aws/aws-sdk-java-v2/pull/3560/

Reply:
Done.